### ISSUE TITLE
False positives by font rendering differences

### ISSUE THREAD

**Issue by tdeekens** [palmerhq/cypress-image-snapshot#18]
Thanks for this great tool. I really enjoy using it. However over time we noticed that font rendering is causing a lot of false positives on our end.

Is there a known way to solve this?

Thanks!

**Comment by cangoektas** [comment]
Haven't tried it myself but I saw this tweet from @jaredpalmer: https://twitter.com/jaredpalmer/status/1012384943912177664

What fonts are you using and for which ones do you get false positives?

**Comment by tdeekens** [comment]
We're using Open Sans. Maybe it also has to do with async loading of fonts. That sometimes the browser applied them and other times now.

I think somehow setting the font to something "simple" before taking the snapshot is nice. Also a bit against the idea as you actually change the visual experience. Where and how would you trigger the font change? 

Thanks for your answer!

**Comment by cangoektas** [comment]
Hmm, yeah true it's definitely a workaround and shouldn't be the recommendation. Maybe you could try to stub the network request for the font loading and `cy.wait` for it before taking the screenshot?

As for injecting CSS, I'm guessing you could start your app with a testing flag and conditionally inject a rule like this: (for example if `NODE_ENV` is set to `test`)

```css
* {
  font-family: Helvetica !important;
}
```

Not really sure though, maybe the Palmer guys can jump in on this 😄

**Comment by tdeekens** [comment]
Thanks! Sorry, we had trouble snapshotting on local dev machines and then CI. Snapshots were consistent across the developer‘s machines but then failed consistently on CI. We already tweaked the sensitivity for that reason but had to a degree where we questioned the over all use of our snapshots. 

Running then on a Docker container feels like a very nice idea. Something I would love to look into. The custom CSS is also a nice idea but also feels like working the problem in a strange undesired way.
